codespell_lib/data/dictionary.txt

```diff
@@ -9983,6 +9986,7 @@ orded->ordered
 orderd->ordered
 orgamise->organise
 organim->organism
+organisaion->organisation
```

Comment:
Can you add the z variant please.

Reply:
Done

Reply:
Could we have the plural too in s and z?

Reply:
Done